pytorch_lightning/trainer/trainer.py

```diff
@@ -297,6 +297,7 @@ def __init__(
         self._device_type = DeviceType.CPU
         self._distrib_type = None
         self._running_stage = None
+        self._predicting = False
```

Comment:
why adding this attrib when you at the same time extend the running stage by Predicting?
```suggestion
```

Reply:
This variable is added to take care of performing prediction in testing part of the evaluation loop. A refactor will be done later on.